launcher/LaunchController.cpp

```diff
@@ -228,6 +228,18 @@ void LaunchController::login() {
                 emitFailed(errorString);
                 return;
             }
+            case AccountState::Disabled: {
+                auto errorString = tr("The launcher's client identification changed. Please remove this account and add it again.");
```

Comment:
Can you make this `has changed`? Without `has` it sounds like it might be of the user's doing.